_includes/navbar.html

```diff
@@ -31,7 +31,7 @@
                 {% assign pg = site.pages | sort: "weight" %}
                 {% for node in pg %}
                     <!-- Neglect imprint.html and index.html in either language -->
-                    {% unless node.path contains "imprint" or node.path contains "index" %}
+                    {% unless node.path contains "imprint" or node.path contains "index" or node.weight <= 0 %}
```

Comment:
What is the weight with non explicitly provided? And if it is zero, couldn’t we get rid of the first two special cases? And if it isn’t, why don’t we set it explicitly to zero for imprint and index and remove the special cases?

Reply:
Good point. Will need to test this.